integr8ly/integreatly-operator
# [INTLY-3894] Use cro to deploy postgres for ups

### Description 
Uses the cloud resource operator to deploy an external database to be used by UPS

`Note: this change requires [this](https://github.com/aerogear/unifiedpush-operator/pull/86) update to the operator (allows external DB connections), which has not been included in the latest release of the ups operator. To work around this a newer image pointing to my quay.io is used until a new version of the operator is released`

### Verification 
- Run the integreatly operator from this branch:
```sh
make cluster/prepare/local
make code/run/service_account
```
- Create an installation cr 
- Ensure that a postgres deployment and connection secret are created in the integreatly namespace
- Ensure that ups is provisioned correctly, all pods are running as expected
